apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -148,11 +148,21 @@ const EVENTS = {
 
   // PL Landing Page
   MY_PL_PAGE_VISITED: 'My Professional Learning Page Visited',
+
+  // Project sharing
+  SHARING_DIALOG_OPEN: 'Sharing_dialog_opened',
```

Comment:
Nit: why are these snake cased and all other events just use spaces?

Reply:
Thanks! Updated.